pkg/converter/sloppy_file_test.go

```diff
@@ -0,0 +1,86 @@
+package converter_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ghodss/yaml"
+	"github.com/google/go-cmp/cmp"
+	"github.com/sloppyio/sloppose/pkg/converter"
+)
+
+var expectedSloppyYml = `project: pkg
+services:
+  apps:
+    busy_env:
+      cmd: sleep 20
+      dependencies:
+      - ../apps/db
+      - ../apps/wordpress
```

Comment:
Wasn't `links` deprecated anyway? If its save to simply ignore it, I'd be fine with that.